Sources/StreamChatUI/ChatChannel/ChatLivestreamChannelVC.swift

```diff
@@ -0,0 +1,498 @@
+//
+// Copyright © 2025 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class ChatLivestreamChannelVC: _ViewController,
```

Comment:
thinking if we even need this one - maybe we just add support to the regular one for livestream channel controller? But only if it's not too much work.

Reply:
What do you mean? Having the livestream controller inside ChatChannelVC?

Reply:
I think it is going to make it very hard to maintain

Reply:
It is going to have a lot of if-statements everywhere, unless we create a sharable interface between them, which could be a good option. For now, I think we should not include any UI out of the box, since the customer is using the controller directly. At most, we should create a dummy UI for the Demo App only.